### PR TITLE
More contrast settings

### PR DESCRIPTION
**Pull request by robert00s**
Close: #2133 
More info: #2133

### CODE REVIEW

File: frontend/ui/widget/configdialog.lua [hunk 1]
```diff
@@ -425,6 +426,34 @@ function ConfigOption:init()
...
+                local buttonprogress_width = Screen:scaleBySize(self.options[c].width or max_buttonprogress_width)
+                local switch = ButtonProgressWidget:new{
+                    width = math.min(max_buttonprogress_width, buttonprogress_width),
+                    height = option_height,
+                    font_face = item_font_face,
+                    font_size = item_font_size,
+                    num_buttons = #self.options[c].values,
+                    position = self.options[c].default_pos,
+                    callback = function(arg)
+                        UIManager:scheduleIn(0.05, function()
```
Comment (Frenzie) [inline frontend/ui/widget/configdialog.lua]: is UIManager:nextTick too fast?

Comment (robert00s) [reply frontend/ui/widget/configdialog.lua]: Right, better use nextTick.

**Comment by Frenzie** [comment]
Awesome, thanks!

**Comment by poire-z** [comment]
I noticed that with this new contrast widget, the screen is not redrawn with the new contrast, at least on my Kobo (on the emulator, there is always a full refresh - and Kindle with its REAGL refresh may have a different and correct behaviour).

But if there:
https://github.com/koreader/koreader/blob/338d14edb78c94bc528baf0bf20c5af1493a9f43/frontend/ui/widget/configdialog.lua#L440-L443
we use `UIManager:scheduleIn(0.05, function() ...` like it's done in the sister function used by the other widgets:
https://github.com/koreader/koreader/blob/338d14edb78c94bc528baf0bf20c5af1493a9f43/frontend/ui/widget/configdialog.lua#L724-L725
it's working as expected!

**Pull request merged**